content/docs/getting-started.md

```diff
@@ -18,109 +18,110 @@ redirect_from:
 
-If you prefer to use your own text editor, you can also [download this HTML file](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), edit it, and open it from the local filesystem in your browser. It does a slow runtime code transformation, so we'd only recommend using this for simple demos.
+اگر آپ اپنا ٹیکسٹ ایڈیٹر استعمال کرنا پسند کرتے ہیں تو ، آپ یہ بھی کرسکتے ہیں [download this HTML file](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), اس میں ترمیم کریں ، اور اسے اپنے براؤزر میں مقامی فائل سسٹم سے کھولیں۔ یہ سست رن ٹائم کوڈ میں تبدیلی کرتا ہے ، لہذا ہم صرف عام ڈیمو کے ل استعمال کرتے ہوئے اس کو استعمال کرنے کی سفارش کرتے ہیں۔
 
 ### Add React to a Website {#add-react-to-a-website}
 
-You can [add React to an HTML page in one minute](/docs/add-react-to-a-website.html). You can then either gradually expand its presence, or keep it contained to a few dynamic widgets.
+آپ کر سکتے ہیں [add React to an HTML page in one minute](/docs/add-react-to-a-website.html). اس کے بعد آپ یا تو آہستہ آہستہ اپنی موجودگی کو بڑھا سکتے ہیں ، یا اسے کچھ متحرک وگیٹس میں رکھ سکتے ہیں.
 
 ### Create a New React App {#create-a-new-react-app}
 
-When starting a React project, [a simple HTML page with script tags](/docs/add-react-to-a-website.html) might still be the best option. It only takes a minute to set up!
+جب کسی رد عمل کا منصوبہ شروع کریں, [a simple HTML page with script tags](/docs/add-react-to-a-website.html) اب بھی بہترین آپشن ہوسکتا ہے۔ اس میں سیٹ اپ میں صرف ایک منٹ لگتا ہے!
 
-As your application grows, you might want to consider a more integrated setup. There are [several JavaScript toolchains](/docs/create-a-new-react-app.html) we recommend for larger applications. Each of them can work with little to no configuration and lets you take full advantage of the rich React ecosystem.
+جب آپ کی ایپلی کیشن میں اضافہ ہوتا ہے ، تو آپ زیادہ مربوط سیٹ اپ پر غور کرنا چاہتے ہیں۔ وہاں ہے [several JavaScript toolchains](/docs/create-a-new-react-app.html) ہم بڑی ایپلی کیشنز کی سفارش کرتے ہیں۔ ان میں سے ہر ایک کم سے کم تشکیل کے ساتھ کام کرسکتا ہے اور آپ کو ری ایکٹ ماحولیاتی نظام سے بھرپور فائدہ اٹھانے دیتا ہے
 
 ## Learn React {#learn-react}
 
-People come to React from different backgrounds and with different learning styles. Whether you prefer a more theoretical or a practical approach, we hope you'll find this section helpful.
+لوگ مختلف پس منظر سے اور سیکھنے کے مختلف اسلوب کے ساتھ اپنا رد عمل ظاہر کرتے ہیں۔ چاہے آپ زیادہ نظریاتی ہوں یا عملی انداز۔
 
-* If you prefer to **learn by doing**, start with our [practical tutorial](/tutorial/tutorial.html).
-* If you prefer to **learn concepts step by step**, start with our [guide to main concepts](/docs/hello-world.html).
+* اگر آپ پسند کریں گے **کر کے سیکھیں**, ہمارے ساتھ شروع کریں [عملی سبق](/tutorial/tutorial.html).
```

Comment:
Some of the translated lines are too literal translations i.e. this line.

```
If you prefer to **learn by doing**
```

into

```
اگر آپ پسند کریں گے **کر کے سیکھیں**
```

Reply:
I think it can be either of following:

```
اگر آپ پسند کریں
```

```
اگر آپ چاہے
```